lib/veri.ml

```diff
@@ -0,0 +1,207 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+module Bap_value = Value
```

Comment:
Don't do this. Don't open `Result` module. And if you will be forced to do this by some evil mastermind, then you should do the opposite:
```
open Bap.Std
open Bil.Result
module Value = Bap.Std.Value
```